l10n_ar_ux/views/account_move_view.xml

```diff
@@ -8,21 +8,13 @@
         <field name="priority">30</field>
         <field name="arch" type="xml">
             <button name="%(account_ux.action_account_change_currency)d" position="after">
-                <!-- distinto margen segun este validado o no (y se vea el boton fa-pencil) -->
-                <span class="oe_inline o_form_label ml-2 mr-2" groups="base.group_multi_currency" attrs="{'invisible': ['|', ('state','=','draft'), ('other_currency', '=', False)]}">-</span>
-                <span class="oe_inline o_form_label mr-2" groups="base.group_multi_currency" attrs="{'invisible': ['|', ('state','!=','draft'), ('other_currency', '=', False)]}">-</span>
+                <span groups="base.group_multi_currency" attrs="{'invisible': [('other_currency', '=', False)]}">(</span>
                 <field name="l10n_ar_currency_rate" attrs="{'invisible': ['|', '|', ('l10n_ar_currency_rate', '=', False), ('l10n_ar_currency_rate', '=', 0.0), ('other_currency', '=', False)]}"/>
                 <field name="computed_currency_rate" string="Currency Rate (preview)" attrs="{'invisible': ['|', ('l10n_ar_currency_rate', '!=', 0.0), ('other_currency', '=', False)]}"/>
```

Comment:
add '|', ('country_code', '!=', 'AR'),  to the invisible domain at computed_currency_rate field

Reply:
done